requirements.txt

```diff
@@ -12,9 +12,12 @@ docopt==0.6.2
 eventbrite==3.3.3
 gunicorn==19.9.0
 idna==2.8
+numpy==1.17.2
```

Comment:
No need to make changes here, just a general comment, there are already python functions to make this kind of stuff with files, maybe it's worth it to take a look if you can do it without adding a library, we can take a look if you want, thanks!